ui/selectors/multichain.ts

```diff
@@ -438,7 +441,15 @@ export function getSelectedAccountMultichainTransactions(
     return undefined;
   }
 
-  return state.metamask.nonEvmTransactions[selectedAccount.id];
+  const transactions = state.metamask.nonEvmTransactions[selectedAccount.id] as
```

Comment:
Nit: The cast on this line, and on line 450, seem to be unnecessary. This is an anti-pattern that we should avoid when we can, see here for reasons why: https://github.com/MetaMask/contributor-docs/blob/main/docs/typescript.md#prefer-type-inference-over-annotations-and-assertions